app/models/record.rb

```diff
@@ -32,13 +32,17 @@ def self.search_by_day!(day, role_id, name, id_branch)
     ).where(
       employees: { branch_id: id_branch }, records: { check_in: date_range }
     )
-    employee = employee.where({ role_id: }) if role_id.present?
+    puts "HERE"
+    puts role_id.present?
```

Comment:
Probably you already know it but you can also use debugger to inspect the data https://edgeguides.rubyonrails.org/debugging_rails_applications.html#entering-a-debugging-session

Reply:
Sorry. I forgot delete this.